config/locales/en.yml

```diff
@@ -2444,6 +2447,8 @@ See the %{link} to find out more about %{sitename}'s features and to start using
     pending: Pending
     shipped: Shipped
   js:
+    "yes": "Yes"
+    "no": "No"
```

Comment:
we don't need double quotes for the keys, right?

Reply:
@sauloperez "yes" and "no" are [reserved](http://yaml.org/type/bool.html) in YAML, and can be [enclosed in quotes as a workaround](https://makandracards.com/makandra/24809-yaml-keys-like-yes-or-no-evaluate-to-true-and-false). :slightly_smiling_face: Interesting quirk. We actually have translations for "(en.)yes" and "(en.)no" earlier in the translations file too, just not under the "js" scope.

Reply:
nice.

I'd put this in context js.products_variants.on_demand so that translators can do their magic if they want. Like in German this will sound better as "on_demand/limited".

Reply:
Thanks, @luisramos0! I've added a commit to translate those strings. They're in `js.variant_overrides.on_demand`, in case the translations for `on_demand` in the variants themselves would be different.

This is back in Code Review.

Reply:
do you still need these js.yes and js.no here?

Reply:
Not anymore, @luisramos0.

Removed in https://github.com/openfoodfoundation/openfoodnetwork/pull/3104/commits/b3f440c17d44cf2721525d00595b434faf36a994. Thanks!